syft/serde.py

```diff
@@ -191,7 +195,7 @@ def _detail_torch_tensor(tensor: bin) -> torch.Tensor:
         cls=tensor,
         torch_tensor=True,
         owner=None,
-        id=None,
+        id=id,
```

Comment:
Should we serialize owners or is that a later concept. (Also do we even need owner for pointer tensor)

Reply:
Also relatedly we should have tests for tensors with None id and non None id

Reply:
We shouldn't ever need to serialize owners because it's always going to be changed to whoever does the deserializing. There might be edge cases but by default I think it makes sense to ignore them